### ISSUE TITLE
frio: Missing checkboxes to select users in Admin

### ISSUE THREAD

**Issue by MrPetovan** [friendica/friendica#3395]
This is self-explanatory, in the frio theme of the user administration page the action buttons to delete and block/unblock are there, but there aren't any checkboxes to select accounts to delete/block/unblock. I had to switch to vier to remove my obsolete feed accounts.

**Comment by rabuzarus** [comment]
Yes, there aren't any Frio specific admin templates at the moment. Someone needs to write it (a good starting point would be the structure of Frios already existent user settings templates).
But we need a solution for the user table you mentioned above. At the moment there is to much information for such a small table. We would need a good GUI concept.

**Comment by fabrixxm** [comment]
I'm on admin aside template... (just to not duplicate work)
small thread hijack:
@rabuzarus I was creating a personalized version of 'admin_aside.tpl', but maybe would be better to rework mod/admin.php to use 'generic_links_widget.tpl' (which, btw, I've personalized for frio, moving the `class="selected"`bit from `a` to containing `li`).
What do you think?

**Comment by rabuzarus** [comment]
Yes why not. The `class="selected"'' should be on the `li` element.
But...
If someone is playing around with the aside widget he/she can have a look if it would be possible to have collapsible menu points. Especially at the admin pages we have many sub categories (e.g at /admin/site).  It would be nice to have it as sub menu entries in the aside widget

**Comment by hoergen** [comment]
Still missing this feature and additionally the screen looks crappy because there is a lot text running out on the right side of the screen

**Comment by MrPetovan** [comment]
@rabuzarus Can we add a wide-column mode for the admin interface? It is a recurrent gripe.

**Comment by MrPetovan** [comment]
A mode where the content column can stretch to the right side of the screen instead of being limited in width.

**Comment by fabrixxm** [comment]
I'm working on this issue

**Comment by rabuzarus** [comment]
hmm, 2 possibilities:
1.) create and use a different page template (`view/theme/frio/php/`)
2.) change the bs class of the column through js

I would vote for 2. but in a different way:
Add an icon for fullscreen at the top of the page and provide a limited list in standard mode. Clicking the fullscreen icon should extend through js the middle column (the one with the user table) to full width (means without left aside and the right free space) and should provide the full user list

Or we put the user table in an xl bs modal

**Comment by rabuzarus** [comment]
One issue will remain. How do we present the table on devices with limited screen width (e.g phones)?

**Comment by MrPetovan** [comment]
I don't think mobile compatibility is important in this case, even with the current display it is unworkable, so we might as well make at least the desktop be usable.

I don't think the left aside should be discarded either.

**Comment by rabuzarus** [comment]
> I don't think the left aside should be discarded either.

Not for default. But in a fullscreen mode which  should be only for real working  with the user table it would be a consistent behaviour (expand in both directions). If someone wants to have the default view (with left aside) he/she can close the fullscreen mode.

**Comment by fabrixxm** [comment]
Something like this (not really mobile)
![admiusers](https://user-images.githubusercontent.com/309361/38942703-c6868c38-432f-11e8-88d2-0e4a4e40920f.png)

**Comment by MrPetovan** [comment]
@rabuzarus I'm not a fan of the JS options, I'd rather have a specific template for the admin interface.

@fabrixxm Is this a mockup? What are the labels "Register date", "Last login" and "Last item"? Are they opening up when one clicks on a user record row?

**Comment by fabrixxm** [comment]
@MrPetovan no, it's code from my local branch.
the user details are toggled on click on "hamburger" icon.
If you can still sort by "Register date", "Last login" or "Last item". The sorted field is shown instead of "Account" column:
<img width="663" alt="schermata 2018-04-18 alle 17 48 31" src="https://user-images.githubusercontent.com/309361/38943056-be18a56c-4330-11e8-8576-d785e4a594c4.png">

**Comment by tobiasd** [comment]
This can be closed I think
